server/methods.js

```diff
@@ -270,7 +270,13 @@ Meteor.methods({
       const instance = Instances.findOne({
         _id: instanceid,
       });
+      var found = mods.find(function(element){
+        return element === instance.admin;
+      });
       if (email === instance.admin) {
+        if(typeof found !== 'undefined') {
+            return [{name: 'owner', type: 'Invalid candidate for moderator', value: instance.admin}];
```

Comment:
`Invalid candidate for moderator` > `An invalid candidate for a moderator`

Reply:
updated